contracts/src/v0.8/ccip/pools/GHO/UpgradeableTokenPool.sol

```diff
@@ -55,6 +58,12 @@ abstract contract UpgradeableTokenPool is IPool, OwnerIsCreator, IERC165 {
     RateLimiter.Config inboundRateLimiterConfig; // Inbound rate limited config, meaning the rate limits for all of the offRamps for the given chain
   }
 
+  /// @dev The storage slot for Proxy Pool address, act as an on ramp "wrapper" post ccip 1.5 migration.
+  /// @dev This was added to continue support for 1.2 onRamp during 1.5 migration, and is stored
+  /// this way to avoid storage collision.
+  // bytes32(uint256(keccak256("ccip.pools.GHO.UpgradeableTokenPool.proxyPool")) - 1)
+  bytes32 internal constant PROXY_POOL_SLOT = 0x75bb68f1b335d4dab6963140ecff58281174ef4362bb85a8593ab9379f24fae2;
```

Comment:
Similar to my other feedback, this should be `POOL_PROXY_SLOT`

Reply:
the self serve pools are referred to as `Proxy Pool` according to the chainlink - [ref](https://github.com/Cyfrin/2024-07-CL-CCIP?tab=readme-ov-file#upgrading-to-self-serve-pools)